EnglishFilter/sections/adservers_firstparty.txt

```diff
@@ -1,6 +1,7 @@
 !
 ! Section contains the list of advertising networks, which are hosted on non advertising sites as subdomains
 !
+||api.adhooah.com^
```

Comment:
Why not `||adhooah.com^$third-party`?

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/91114/commits/7e739e1e438eca0c3a5b3320a266182c0d1372c0